### PR TITLE
Fix UNICODE_DOT, codegen-units=2, add .editorconfig

### PR DESCRIPTION
**Pull request by serprex**

### CODE REVIEW

File: src/curses/consts.rs [hunk 1]
```diff
@@ -3,15 +3,15 @@ use ncurses::chtype;
...
-pub const UNICODE_DOT: chtype = '\u{2219}' as chtype;
+pub const WALL_CH: char = '#';
+pub const DOOR_OPEN_CH: char = '_';
+pub const DOOR_CLOSED_CH: char = '×';
+pub const STATUE_CH: char = '&';
+pub const STAIRS_DOWN_CH: char = '>';
+pub const WATER_CH: char = '~';
+pub const NOTHING_CH: char = '~';
+pub const NORMAL_DOT: char = '.';
+pub const UNICODE_DOT: char = '\u{2022}';
```
Comment (CryZe) [inline src/curses/consts.rs:14]: Why is this escaped?

Comment (serprex) [inline src/curses/consts.rs:14]: It likely doesn't need to be. There are a number of centered-circle unicode characters tho & ascii purity

Comment (CryZe) [inline src/curses/consts.rs:14]: The DOOR_CLOSED_CH should be escaped too then. But I still think that just putting the Unicode character there is fine.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+root = true
```
Comment (dpc) [inline .editorconfig:1]: Today I've learned.

**Pull request merged**